pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -106,16 +124,18 @@ var _ = Describe("Pod Network", func() {
 			Mtu:     1410,
 			Gateway: gw}
 
-		masqueradeGwStr = "10.0.2.1/30"
+		ipv4Cidr = testParams["IPV4"].cidr
+		masqueradeGwStr = testParams["IPV4"].samples["GW"]
 		masqueradeGwAddr, _ = netlink.ParseAddr(masqueradeGwStr)
 		masqueradeGwIp = masqueradeGwAddr.IP.String()
-		masqueradeVmStr = "10.0.2.2/30"
+		masqueradeVmStr = testParams["IPV4"].samples["VM"]
```

Comment:
if we're just setting values on variables once in this before function, i'm not seeing the value in all these maps and everything.

Reply:
I removed this struct.